cpu/esp8266/Makefile.include

```diff
@@ -143,5 +147,5 @@ else
     export FFLAGS += -p $(PORT) -b $(PROGRAMMER_SPEED) write_flash
     export FFLAGS += -fm $(FLASH_MODE)
     export FFLAGS += 0 $(ELFFILE)-0x00000.bin
-    export FFLAGS += 0x10000 $(ELFFILE)-0x10000.bin
+    export FFLAGS += 0x10000 $(ELFFILE)-0x10000.bin; esptool.py -p $(PORT) run
```

Comment:
this looks a bit weird, can you explain. To me this looks like the latter is a necessary POSTFLASH command?

Reply:
Yep, it is the comment to reset after flashing. I couldn't figure out any other way to define something like that. RESET variable is not doing that since it requires to call `make reset` explicitly.

Reply:
okay, then leave as is for now. But we should keep this in mind and clean it up later on - meaning: this should be handled properly by the build system.